src/bci_build/package/python.py

```diff
@@ -48,7 +48,7 @@ def _get_python_kwargs(py3_ver: _PYTHON_VERSIONS, os_version: OsVersion):
 
     kwargs = {
         "name": "python",
-        "pretty_name": f"Python {py3_ver} development",
+        "pretty_name": f"Python {py3_ver} Development",
```

Comment:
no we use sentence style capitalization here. see https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-capitalization-sentence

Reply:
we're a bit inconsistent, filed https://github.com/SUSE/BCI-dockerfile-generator/pull/1272 as a followup

Reply:
Yes, but many of the READMEs are "Name Container Image".

We should change all READMEs in that case to be lowercase.
Otherwise, without this change, the container title will be "Python development Container Image", which is odd.

Reply:
Fixed to be all sentence style.